app/src/main/java/org/oppia/app/player/audio/AudioFragmentPresenter.kt

```diff
@@ -48,6 +58,8 @@ class AudioFragmentPresenter @Inject constructor(
 
   /** Sets up SeekBar listener, ViewModel, and gets VoiceoverMappings or restores saved state */
   fun handleCreateView(inflater: LayoutInflater, container: ViewGroup?): View? {
+    val internalProfileId = fragment.arguments!!.getInt(AUDIO_FRAGMENT_PROFILE_ID_ARGUMENT_KEY, -1)
```

Comment:
To keep code consistency, we should fetch the id in `onCreateView` function of `AudioFragment` and pass it to the presenter via `handleCreateView` function. Checkout `StoryFragment` and its presenter.

Reply:
I did it keeping state fragment implementation in mind and it is consistent  with the implementation in StateFragmentPresenter PTAL

Reply:
Done.